bin/picca_Pk1D.py

```diff
@@ -272,7 +235,7 @@ def process_all_files(index_file_args):
                     selection = (((k > args.kmin_noise_avg) if args.kmax_noise_avg is not None else 1) &
                                  ((k < args.kmax_noise_avg) if args.kmax_noise_avg is not None else 1))
                 mean_pk_noise = np.mean(pk_noise[selection])
-                pk = (pk_raw - pk_noise) / correction_reso
+                pk = (pk_raw - mean_pk_noise) / correction_reso
```

Comment:
this is now switching the defaults from taking the bin wise noise power to taking the mean, so would be good to have that documented (except if we've used the mean for all previous analyses, just not as the default here)